pkg/codegen/dotnet/importer.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dotnet
+
+import (
+	"encoding/json"
+
+	"github.com/pulumi/pulumi/pkg/v2/codegen/schema"
+)
+
+// CSharpPropertyInfo represents the C# language-specific info for a property.
+type CSharpPropertyInfo struct {
+	Name string `json:"name,omitempty"`
+}
+
+// CSharpPackageInfo represents the C# language-specific info for a package.
+type CSharpPackageInfo struct {
+	PackageReferences map[string]string `json:"packageReferences,omitempty"`
+	Namespaces        map[string]string `json:"namespaces,omitempty"`
+}
+
+// Importer implements schema.Language for .NET.
+var Importer schema.Language = importer(0)
+
+type importer int
```

Comment:
Any reason to make this an `int` rather than an empty struct?

Reply:
I see `int` a bit more frequently for this sort of stateless interface.